src/test/java/org/cactoos/scalar/package-info.java

```diff
@@ -29,9 +29,6 @@
  *  Assertion from cactoos-matchers. Once there is no more usage of
  *  MatcherAssert.assertThat, add the signature of MatcherAssert.assertThat to
  *  forbidden-apis.txt
- * @todo #1367:30min Replace all the IsEqual with ScalarHashValue in Scalar
```

Comment:
@marceloamadeu have you verified there is really no IsEqual incorrectly used in the tests of `scalar` package?